cmd/heimdalld/main.go

```diff
@@ -98,12 +102,14 @@ func GetSignerInfo(pub crypto.PubKey, priv []byte, cdc *codec.Codec) ValidatorAc
 
 func main() {
 	cdc := app.MakeCodec()
-	ctx := server.NewDefaultContext()
+	serverCtx := server.NewDefaultContext()
```

Comment:
This variable change is creating a lot of new code.

Reply:
This is passed to only two functions from the main itself, not sure I get this comment.

Also, if we could merge server context and standard context with context.WithValue, where server context would be nested.